### DISCUSSION TITLE
Im not able to use my gpu on blender

### DISCUSSION THREAD

**Discussion by Izaird** [rocm-arch/rocm-arch#973] [Support]
When I open the settings menu and I try to select hip this message appears:
No compatible GPU found for cycles

my gpu is a rx 6750 xt 

I have installed on my pc this packages:
rocm-hip-sdk
rocm-opencl-sdk
rocm-hip-runtime
hip-runtime-amd

**Comment by acxz** [comment]
This discussion should be posted over at the blender PKGBUILD issues page, since `blender` (and the associated `rocm` packages for it) is in the `[extra]` repo.

https://gitlab.archlinux.org/archlinux/packaging/packages/blender/-/issues

Please post your issue there. Closing this discussion here.

Closing this issue here as
